dali/kernels/imgproc/color_manipulation/hsv_cpu_test.cc

```diff
@@ -0,0 +1,184 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <gtest/gtest.h>
+#include <vector>
+#include <tuple>
+#include "dali/kernels/test/tensor_test_utils.h"
+#include "dali/kernels/test/kernel_test_utils.h"
+#include "dali/kernels/imgproc/color_manipulation/hsv_cpu.h"
+#include "dali/kernels/imgproc/color_manipulation/color_manipulation_test_utils.h"
+
+namespace dali {
+namespace kernels {
+namespace hsv {
+namespace test {
+
+namespace {
+
+static constexpr int ndims = 3;
+
+void fill_roi(Box<2, int> &roi) {
+    roi = {{1, 2},
+           {5, 7}};
+}
+
+
+template<class GtestTypeParam>
+using TheKernel = HsvCpu<typename GtestTypeParam::Out, typename GtestTypeParam::In>;
+
+}  // namespace
+
+
+template<class InputOutputTypes>
+class HsvCpuTest : public ::testing::Test {
+ protected:
```

Comment:
to avoid typing `typename` all the time you can do here something like:
```
using In = typename InputOutputTypes::In;
```
and so on.

Then you only use `In` and you can get rid of `TheKernel`

Reply:
Done